src/care/PluginData.h

```diff
@@ -0,0 +1,64 @@
+#ifndef _CARE_PluginData_H_
+#define _CARE_PluginData_H_
+
+#include "chai/PointerRecord.hpp"
+#include "care/config.h"
+#include <vector>
+#include <functional>
+#include <unordered_map>
+
+namespace chai {
+   struct PointerRecord;
+}
+
+namespace care{
+
+	class PluginData	{
+		public:
+			PluginData() = default;
+
+         static void setFileName(const char * name);
```

Comment:
I think that the indentation is inconsistent here, between PluginData() = default; and the functions below